lint/base_linter/node_linter.py

```diff
@@ -146,11 +147,45 @@ def find_local_executable(self, start_dir, npm_name):
                     self.notify_failure()
                     raise linter.PermanentError()
 
-                # A 'package.json' not yet installed?
+                # No 'node_modules' folder ?
                 if not os.path.exists(os.path.join(path, 'node_modules', '.bin')):
                     is_dep = bool(manifest.get('dependencies', {}).get(npm_name))
                     is_dev_dep = bool(manifest.get('devDependencies', {}).get(npm_name))
                     if is_dep or is_dev_dep:
```

Comment:
I think we can immediately have the side effect of setting project_root here bc we have found the right package.json.